web/src/components/planner/PlannerFactorySatisfaction.vue

```diff
@@ -30,12 +30,20 @@
         :model-value="showSatisfactionBreakdowns"
         @change="changeSatisfactionBreakdowns"
       />
+      <v-switch
+        v-model="hideInternalOutputs"
+        class="ml-4"
+        color="primary"
+        hide-details
+        label="Hide Internal Products"
```

Comment:
```suggestion
        label="Hide Internal Products & Raw"
```
Your code currently also hides Raw, which in this context does make sense, so just adjusting the label to match the behaviour.